openssl.yaml

```diff
@@ -12,14 +12,6 @@ package:
     runtime:
       - openssl-provider-legacy
 
-vars:
-  fips-enable: 'no-fips'
-
-options:
-  fips:
-    vars:
-      fips-enable: 'enable-fips'
```

Comment:
This might be the last remaining usage of `options` in our repos, which means we might be able to remove this from Melange 👀

Reply:
yay!